server/tournaments/index.ts

```diff
@@ -379,7 +379,7 @@ export class Tournament extends Rooms.RoomGame {
 			return;
 		}
 
-		const gameCount = user.games.size;
+		const gameCount = [...Rooms.rooms.values()].filter(r => !(r.game && r.game.playerTable[user.id])).length;
```

Comment:
```suggestion
		const gameCount = user.getGames().length;
```

Reply:
13 days ago, you resolved this, and you _still_ haven't committed this change. :[

Reply:
I checked my force-pushes, I didn't accidentally revert these. You just never committed any of them.